eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxow.hs

```diff
@@ -219,9 +215,8 @@ instance
       2 -> SumD MissingRequiredDatums <! From <! From
       3 -> SumD NotAllowedSupplementalDatums <! From <! From
       4 -> SumD PPViewHashesDontMatch <! From
-      5 -> SumD MissingRequiredSigners <! From
-      6 -> SumD UnspendableUTxONoDatumHash <! From
-      7 -> SumD ExtraRedeemers <! From
+      5 -> SumD UnspendableUTxONoDatumHash <! From
+      6 -> SumD ExtraRedeemers <! From
       n -> Invalid n
```

Comment:
Ditto. If the skipped number bothers you, feel free to add a comment or even better a decoder failure with a better message.
```suggestion
      5 -> SumD id <! fail "MissingRequiredSigners was deprecated"
      6 -> SumD UnspendableUTxONoDatumHash <! From
      7 -> SumD ExtraRedeemers <! From
      n -> Invalid n
```

Reply:
I think this is good enough as it clearly indicates that nobody should be looking for `MissingRequiredSigners` anymore. :) Thanks!